src/reducers/appStateReducer.js

```diff
@@ -0,0 +1,27 @@
+import * as actions from "../actions";
+import { login, logout } from "../api";
+
+const appStateReducer = (state = { isLoggedIn: false }, action) => {
+   const { type, payLoad } = action;
+
+   switch (type) {
+      case actions.LOGIN: {
+         login(payLoad.email, payLoad.password);
+         return { ...state, isLoggedIn: true };
+      }
+
+      case actions.LOGOUT: {
+         logout();
+         return { ...state, isLoggedIn: false };
+      }
+
+      case actions.REGISTER: {
+         return { ...state, isLoggedIn: false };
```

Comment:
When user Register we going to login him to the system right away. So isLoggedIn = true after register success

Reply:
done